package.json

```diff
@@ -4,20 +4,17 @@
   "description": "Create API error objects for DADI applications",
   "main": "index.js",
   "scripts": {
-    "test": "standard 'lib/*.js' | snazzy && env NODE_ENV=test ./node_modules/.bin/istanbul cover --report cobertura --report text --report html --report lcov ./node_modules/mocha/bin/_mocha test",
-    "posttest": "./scripts/coverage.js && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "semantic-release": "semantic-release",
+    "test": "standard 'lib/*.js' | snazzy && env NODE_ENV=test ./node_modules/.bin/nyc --reporter=lcov ./node_modules/mocha/bin/_mocha test && ./node_modules/.bin/nyc report",
+    "posttest": "./scripts/coverage.js",
     "travis-deploy-once": "travis-deploy-once"
```

Comment:
these travis commands can go as well, they are inserted by the semantic release setup